### DISCUSSION TITLE
huge cache_items table in size

### DISCUSSION THREAD

**Discussion by LeaveAirykson** [pimcore/pimcore#11898] [Q&A / Support]
hello there.

We run a website with pimcore 10.3.1 and while monitoring our MySQL DBs we saw that the connected db reached a size of 18GB. 16.7GB alone reside in the cache_items table. this raised some questions I want to adress here.

1. Is this (considering the amount of documents, editables, etc.) a "normal" size for the cache_items table or not?
2. At which point in time and through which process are the records in the cache_items table dumped/renewed?
3. Should the maintenance cronjob (`php ~/web/current/bin/console pimcore:maintenance`)  , which runs every 15 minutes, free up space in the cache_items table or not?

I'm happy for every push in the right direction and/or help.

Some db table stats:
| table               | records |   size |
|---------------------|--------:|-------:|
| cache_items         | 157.571 | 16,7GB |
| documents_editables |  11.112 |  3,2MB |
| dependencies        |   3.808 |  416KB |
| versions            |   3.045 |  4,5MB |
| properties          |    2048 |  4,5MB |
| documents           |     238 |  128KB |
| objects             |      58 |  144KB |

cheers!

**Comment by troxxs** [comment]
I have the same problem on pimcore 11.3, the cache_items table is getting bigger and bigger until i clear it. it was up to 90GB. the maintenance cron job is running every 5 minutes. So can someone please respond to this question or @LeaveAirykson, did you maybe found the reason or a solution for this?

thank you

**Reply by dpfaffenbauer** [reply]
move the cache to redis, give you better performance too. Redis will auto clean old keys based on configuration as well.

**Reply by jdreesen** [reply]
Filesystem/DB caches do not automatically prune stale items.

Pimcore has a maintenance task for it (since `v11.0.12`), so make sure it's executed: https://github.com/pimcore/pimcore/pull/15950

Apart from that, there's also `php bin/console cache:pool:prune` from Symfony which should do the same: https://symfony.com/doc/6.4/components/cache/cache_pools.html#pruning-cache-items
